pkg/registry/apps/statefulset/storage/storage_test.go

```diff
@@ -42,6 +43,7 @@ func newStorage(t *testing.T) (*REST, *StatusREST, *etcdtesting.EtcdTestServer)
 // createStatefulSet is a helper function that returns a StatefulSet with the updated resource version.
 func createStatefulSet(storage *REST, ps apps.StatefulSet, t *testing.T) (apps.StatefulSet, error) {
 	ctx := genericapirequest.WithNamespace(genericapirequest.NewContext(), ps.Namespace)
+	ctx = genericapirequest.WithRequestInfo(ctx, testhelper.FakeRequestInfo("", ""))
```

Comment:
I wouldn't expect to need to do this in all the tests... I'd expect the store to use the requestinfo from the context if present, otherwise use the default qualified resource